docs/getting-started/run-applications.md

```diff
@@ -33,12 +33,38 @@ on-demand by Crossplane.
 > published a PostgreSQLInstance with at least one working Composition in order
 > to create the OAM application we'll use in this guide.
 
-## Install the OAM Addon
+## Infrastructure Operator
+
+### Install OAM build-in workloads and traits
 
 As the infrastructure operator our work is almost done - we defined, published,
 and composed the infrastructure  that our application developer and operator
-team-mates will use in the previous guide. One task remains for us, which is to
-install and configure Crossplane's OAM addon.
+team-mates will use in the previous guide. One task remains is, we must define 
+the [_workloads_] and [_traits_] that our platform supports. We'll discuss workloads 
+and traits in more detail later.
+
+The infrastructure operator could choose which workloads
+and traits their platform supports by creating or deleting `WorkloadDefinitions`
+and `TraitDefinitions` like below:
+
+```yaml
+---
+apiVersion: core.oam.dev/v1alpha2
+kind: WorkloadDefinition
+metadata:
+  name: containerizedworkloads.core.oam.dev
+spec:
+  definitionRef:
+    name: containerizedworkloads.core.oam.dev
+``` 
+Run the following command to add support for the build-in workloads and traits required
+by this guide:
```

Comment:
This should be "built-in", not "build-in". I suggest we remove or rephrase "built-in" though - it's not clear to me what it means in the context of this guide.

Reply:
These controllers are for [core](https://github.com/oam-dev/spec/blob/master/3.workload.md#core-workload) and [standard](https://github.com/oam-dev/spec/blob/master/3.workload.md#standard-workload) OAM workloads/traits/scopes. Feel free to find a better word or I will just use core and standard instead.

Reply:
The controllers are, but keep in mind we're also adding a workload definition for `MySQLInstanceRequirement`, which is not built-in/core/standard.

Reply:
> The controllers are,

The controllers are what we installed below, no?

Reply:
Oh, built-in should be deleted from this line. The built-in is for name of controllers.